lib/skunk/cli/commands/compare.rb

```diff
@@ -37,7 +37,9 @@ def build_details
       end
 
       def build_details_path
-        "#{::RubyCritic::Config.compare_root_directory}/build_details.txt"
+        compare_root_directory = ::RubyCritic::Config.compare_root_directory
+        FileUtils.mkdir_p(compare_root_directory) unless File.exist?(compare_root_directory)
```

Comment:
@sebastian-palma Could you move this line to the `build_details` method? `build_details_path` should be a read-only method that returns the path. `build_details` should write to that file *and* create the compare root directory if it doesn't exist.

Reply:
Yeah, no problem @etagwerker. Are you comfortable with moving the `details` local variable to a method?

If I add the change you propose we're going to have a Reek _TooManyStatements_ smell on `build_details`.

Reply:
@sebastian-palma Yes, that is a great idea 👍

Reply:
I ended up moving the creation of the folder to a separate class under its own method because moving it to a method on the `Compare` class would result in a Utility Function warning.

Still I don't know if that's ok or if we can do something better @etagwerker  (maybe also with the naming).